core/src/main/java/jenkins/security/ClassFilterImpl.java

```diff
@@ -76,6 +80,11 @@ public static void register() {
             return;
         }
         ClassFilter.setDefault(new ClassFilterImpl());
+        if (SUPPRESS_ALL) {
+            LOGGER.warning("All class filtering suppressed. Your Jenkins installation is at risk from known attacks. See https://jenkins.io/redirect/class-filter/");
```

Comment:
That blog post does not describe the blacklist concerns, but we can update the link later